vue.config.js

```diff
@@ -1,4 +1,24 @@
 const { defineConfig } = require('@vue/cli-service')
+
+// Map envs with VUE_APP_ prefix only if CONTENTSTACK_ variable is undefined for launch integration
+(function () {
+  if (typeof process.env.CONTENTSTACK_API_KEY === 'undefined') {
```

Comment:
we would want to check for `process.env.VUE_APP_CONTENTSTACK_API_KEY` and not `process.env.CONTENTSTACK_API_KEY` and similar for all if statements